tools/packaging/static-build/cache_components_main.sh

```diff
@@ -29,10 +29,23 @@ cache_firecracker_artifacts() {
 
 cache_kernel_artifacts() {
 	local kernel_tarball_name="kata-static-${KERNEL_FLAVOUR}.tar.xz"
+	if [[ ${TEE} == "sev" ]]; then
```

Comment:
Are we currently setting TEE?  Or can this be infered from the "KERNEL_FLAVOUR"?
If it's not yet used, and KERNEL_FLAVOUR could be used instead, please use KERNEL_FLAVOUR.

Reply:
We are currently setting it in CCv0 branch. But I can use KERNEL_FLAVOR.